save_search_page/README.md

```diff
@@ -0,0 +1,11 @@
+# Save Current Search Results Page widget #
+
+# Description and Use #
+
+This widget is designed to be used on the search results page within Watson Explorer Application Builder.
+
+The intention is to save the current url to the current_user.properties in zookeeper. We create a new store called current_user.properties["saved_searches"]. This property accepts a key/value pair. The key is a lable defined by the user and the value is always the applications current url. Currently if the user enters a key that already exists the current entry is updated.
```

Comment:
save the current url => save the current URL (capitalization)
The key is a lable => The key is a label (spelling)
applications current url => application's current URL (apostrophe & capitalization)
Currently if the user enters a key that... => If the user enters a key that... (remove 'currently')
that already exists the current => that already exists, the current (add comma)